### DISCUSSION TITLE
Error while making clicker app

### DISCUSSION THREAD

**Discussion by Ryan-Tung** [sst-inc/help#8] [Q&A - iOS] [answered]
iOS Session 3
<img width="674" alt="Screenshot 2021-07-27 at 5 20 25 PM" src="https://user-images.githubusercontent.com/88029861/127129760-ed5cc773-1fa6-493d-8359-41cf07c986d3.png">

When I try to build it fails.
It says "Build Failed"

**Comment by granwyntan** [comment]
Hi, could u try selecting "None" for your team, if not, mind sending your source code or an entire screenshot or video?
<img width="201" alt="Screenshot 2021-07-27 at 20 10 32" src="https://user-images.githubusercontent.com/61010627/127151270-0256b321-4117-403f-b437-5208f4e343cf.png">
Thanks

**Reply by Ryan-Tung** [reply]
I don't know how to go to change my teams and this is my source code.
[Clicker App.zip](https://github.com/sst-inc/help/files/6890442/Clicker.App.zip)

**Reply by DeeYuan1** [reply]
[Clicker bot.zip](https://github.com/sst-inc/help/files/6892483/Clicker.bot.zip)
i also have the same problem

**Reply by granwyntan** [reply]
OK, I'll try help

**Reply by DeeYuan1** [reply]
i managed to fix it

**Reply by granwyntan** [reply]
Nice

**Comment by granwyntan** [comment, marked as answer]
https://user-images.githubusercontent.com/61010627/127728101-cc2df26a-4c38-4451-830d-52577b8a27b5.mov

@Ryan-Tung go to Project Settings > Signing and Capabilities, select Team as None, if you see an error like below, it's perfectly fine
<img width="314" alt="Screenshot 2021-07-31 at 12 06 06" src="https://user-images.githubusercontent.com/61010627/127728135-302bcab6-6329-4610-a571-7f99e9eac270.png">
